common/service.go

```diff
@@ -11,30 +11,84 @@ type (
 )
 
 const (
-	EmptyService Service = iota
-	StarWarsService
-	BaseService
-	BTCService
-	ETHService
-	GAIAChainRPCArchiveService
+	EmptyService    Service = iota
+	StarWarsService Service = 1
+	BTCService      Service = 10
+	ETHService      Service = 20
 )
 
 var ServiceLookup = map[string]int32{
-	"unknown":   0,
-	"swapi.dev": 1, // star wars API for development purposes
-	"github.com/arkeonetwork/arkeo-mainnet-fullnode": 2,
-	"btc-mainnet-fullnode":                           3,
-	"eth-mainnet-fullnode":                           4,
-	"gaia-mainnet-rpc-archive":                       5,
+	"unknown":                      0,
+	"swapi.dev":                    1, // star wars API for development purposes
+	"arkeo-mainnet-fullnode":       2,
+	"avax-mainnet-fullnode":        3,
+	"avax-mainnet-archivenode":     4,
+	"bch-mainnet-fullnode":         5,
+	"bch-mainnet-lightnode":        6,
+	"bnb-mainnet-fullnode":         7,
+	"bsc-mainnet-fullnode":         8,
+	"bsc-mainnet-archivenode":      9,
+	"btc-mainnet-fullnode":         10,
+	"btc-mainnet-lightnode":        11,
+	"cardano-mainnet-relaynode":    12,
+	"cosmos-mainnet-fullnode":      13,
+	"doge-mainnet-fullnode":        14,
+	"doge-mainnet-lightnode":       15,
+	"etc-mainnet-archivenode":      16,
+	"etc-mainnet-fullnode":         17,
+	"etc-mainnet-lightnode":        18,
+	"eth-mainnet-archivenode":      19,
+	"eth-mainnet-fullnode":         20,
+	"eth-mainnet-lightnode":        21,
+	"ltc-mainnet-fullnode":         22,
+	"ltc-mainnet-lightnode":        23,
+	"optimism-mainnet-fullnode":    24,
+	"osmosis-mainnet-fullnode":     25,
+	"polkadot-mainnet-fullnode":    26,
+	"polkadot-mainnet-lightnode":   27,
+	"polkadot-mainnet-archivenode": 28,
+	"polygon-mainnet-fullnode":     29,
+	"polygon-mainnet-archivenode":  30,
+	"sol-mainnet-fullnode":         31,
+	"thorchain-mainnet-fullnode":   32,
+	"unchained-production":         33,
```

Comment:
what is unchained-production? Is it one daemon for all chains (rather than one daemon for one chain)?